### PR TITLE
add eCash

### PR DESCRIPTION
**Pull request by mishfit**
Relies on https://github.com/dgarage/NBXplorer/pull/369

### CODE REVIEW

File: BTCPayServer.Common/Altcoins/BTCPayNetworkProvider.ECash.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<<<<<<< HEAD
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+=======
+using System.Collections.Generic;
+>>>>>>> 8532ad69 (add ecash)
```
Comment (Kukks) [inline BTCPayServer.Common/Altcoins/BTCPayNetworkProvider.ECash.cs:7]: conflict pointers

**Review by Kukks — CHANGES_REQUESTED** [review]
Your logo is probably not in the corrected dimensions as it usually fits centered in a qr code.

**Review by Kukks — CHANGES_REQUESTED** [review]
The way this PR stands and how the PR dependency looks, I don't think it will be mergeable unless you go back and add the coin to NBitcoin the normal way without all the changes. As nicolas mentioned before, we won't include dedicated nuget packages for altcoins as that presents attack vectors.

I will close this PR for now, reopen, when you have finished NBitcoin + NBXplorer support

**Comment by Inyourdreams182** [comment]
> Relies on [dgarage/NBXplorer#369](https://github.com/dgarage/NBXplorer/pull/369)

U want to forces to me

**Pull request closed without merging**